comtypes/tools/codegenerator.py

```diff
@@ -345,6 +346,11 @@ def ArrayType(self, tp):
     _enumvalues = 0
     def EnumValue(self, tp):
         value = int(tp.value)
+        if keyword.iskeyword(tp.name):
+            # XXX use logging!
+            if __debug__:
```

Comment:
How does [that](https://github.com/laripk/comtypes/commit/81f65d9bf98449345100404e944b2ef835a8665e) look?